runsc/boot/config.go

```diff
@@ -138,6 +138,9 @@ type Config struct {
 	// Overlay is whether to wrap the root filesystem in an overlay.
 	Overlay bool
 
+	// fsGoferHostUDSAllowed enables the gofer to mount a host UDS
```

Comment:
Comments should begin with the name of the thing being described and end in a period. Please fix both.

https://github.com/golang/go/wiki/CodeReviewComments#comment-sentences